### PR TITLE
Functional tests : Migration TS (Migrate `@utils/files`)

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Functional tests : Migration TS (Migrate `@utils/files`)
| Type?             | improvement
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no no
| Fixed ticket?     | Relative to  #30263
| Related PRs       | N/A
| How to test?      | Nightly is :green_circle: : https://github.com/Progi1984/ga.tests.ui.pr/actions/runs/3732686782
| Possible impacts? | N/A

### CODE REVIEW

**Review by SD1982 — DISMISSED** [review]

**Review by boubkerbribri — DISMISSED** [review]

**Review by cfarhani06 — DISMISSED** [review]

**Review by matthieu-rolland — COMMENTED** [review]
A rebase is needed @Progi1984

File: tests/UI/campaigns/functional/BO/11_international/01_localization/02_languages/02_CRUDLanguage.ts [hunk 1]
```diff
@@ -15,6 +15,16 @@ import addLanguagePage from '@pages/BO/international/languages/add';
...
+require('module-alias/register');
+// Using chai
+const {expect} = require('chai');
+
+// Import pages
+const dashboardPage = require('@pages/BO/dashboard');
+const localizationPage = require('@pages/BO/international/localization');
+const languagesPage = require('@pages/BO/international/languages');
+const addLanguagePage = require('@pages/BO/international/languages/add');
+
```
Comment (boubkerbribri) [inline tests/UI/campaigns/functional/BO/11_international/01_localization/02_languages/02_CRUDLanguage.ts]: You should use imports as it's a `ts` file

Comment (Progi1984) [reply tests/UI/campaigns/functional/BO/11_international/01_localization/02_languages/02_CRUDLanguage.ts]: Fixed. Sorry, bad rebase.

**Review by boubkerbribri — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]
🍪

**Pull request merged**